src/common-utils/main.sh

```diff
@@ -385,6 +385,11 @@ else
     user_rc_path="/home/${USERNAME}"
 fi
 
+if [ ! -d "${user_rc_path}" ]; then
+    mkdir -p "${user_rc_path}"
+    chown ${USERNAME}:${group_name} "${user_rc_path}"
+fi
+
```

Comment:
Can you bump the patch version? Thanks

https://github.com/devcontainers/features/blob/1a3145d6c52a6b8d319a5639acdcff408f721c67/src/common-utils/devcontainer-feature.json#L3